### PR TITLE
Content Modification- Audit Guideline addition

### PR DESCRIPTION
**Pull request by rishuranjanofficial**
Audit Guideline addition

### CODE REVIEW

File: pages/attacks/Content_Spoofing.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 layout: col-sidebar
 title: Content Spoofing
 author: 
-contributors: 
+contributors: [Rishu Ranjan](https://github.com/rishuranjanofficial/)
```
Comment (kingthorin) [inline pages/attacks/Content_Spoofing.md:6]: Just names.

File: pages/attacks/Content_Spoofing.md [hunk 2]
```diff
@@ -51,6 +53,13 @@ followers and distribute misleading Content Spoofing payload via
 Twitter/Facebook/Instagram/ similar popular channel. This will lead
 media to assume news is correct and create headline stories.
 
+## Audit Guideline
+
+Text injection can be easily found where
+- User input via parameter or directly in the URL is reflected in the page response.
```
Comment (kingthorin) [inline pages/attacks/Content_Spoofing.md:59]: if

File: pages/attacks/Content_Spoofing.md [hunk 3]
```diff
@@ -51,6 +53,13 @@ followers and distribute misleading Content Spoofing payload via
 Twitter/Facebook/Instagram/ similar popular channel. This will lead
 media to assume news is correct and create headline stories.
 
+## Audit Guideline
+
+Text injection can be easily found where
```
Comment (kingthorin) [inline pages/attacks/Content_Spoofing.md:58]: Colon at the end

File: pages/attacks/Content_Spoofing.md (hunk 2 above)
Comment (kingthorin) [reply pages/attacks/Content_Spoofing.md:59]: Should be consistent in using (or not) periods for bullets

File: pages/attacks/Content_Spoofing.md (hunk 1 above)
Comment (rishuranjanofficial) [reply pages/attacks/Content_Spoofing.md]: Done

File: pages/attacks/Content_Spoofing.md (hunk 3 above)
Comment (rishuranjanofficial) [reply pages/attacks/Content_Spoofing.md]: added

File: pages/attacks/Content_Spoofing.md (hunk 2 above)
Comment (rishuranjanofficial) [reply pages/attacks/Content_Spoofing.md]: is it fine?

**Pull request merged**